### ISSUE TITLE
ggMarginal grouped boxplots are different widths

### ISSUE THREAD

**Issue by nathanieldelaneybusch** [daattali/ggExtra#169]
Marginal boxplots colored by groups produce boxes of different widths. Using params lists (e.g. xparams = list(varwidth = FALSE)) does not appear to fix the issue.

Example:
```
scatterplotMPG = ggplot( data=mpg, aes(x=cty, y = hwy)) +
  geom_point(data=mpg[mpg$drv%in%c('r', 'f'), ], aes(color=factor(drv)), alpha=0.5) +
  geom_point(data=mpg[mpg$drv%in%c('4'), ], color='gray', alpha=0.3)+
  theme(legend.position = "bottom")

ggExtra::ggMarginal(scatterplotMPG, type = "boxplot",
                    groupFill = TRUE, groupColour = TRUE, size=7,
                    xparams = list(varwidth = FALSE),
                    yparams = list(varwidth = FALSE))
```
![image](https://user-images.githubusercontent.com/35230369/198736362-d986acea-98d6-42e5-ba5a-f6a29d024cf3.png)

I am using ggExtra 0.10.0 and ggplot2 3.3.5 on a windows machine.

**Comment by crew102** [comment]
Hey @nathanieldelaneybusch , sorry for the delay, and thanks for the reproducible example. I'll open up a PR soon with a fix.

**Comment by daattali** [comment]
fixed with #173 -- thanks to @crew102
